index.html

```diff
@@ -463,20 +460,20 @@
           <li>Depending on <var>orientation</var> value, do the following:
             <dl>
               <dt>
-                <code><dfn for=
-                'OrientationLockType'>portrait-primary</dfn></code> or
-                <code><dfn for=
-                'OrientationLockType'>portrait-secondary</dfn></code> or
-                <code><dfn for=
-                'OrientationLockType'>landscape-primary</dfn></code> or
-                <code><dfn for=
-                'OrientationLockType'>landscape-secondary</dfn></code>
+                <dfn data-dfn-for=
```

Comment:
as above... then you don't need `data-dfn-for` here... ReSpec always looks for the nearest section with "data-dfn-for=".